qml/PolarGrid.qml

```diff
@@ -0,0 +1,107 @@
+import QtQuick 2.12
+import QtQuick.Shapes 1.12
+
+Item {
+    id: root
+    width: 400
+    height: width
+
+    Shape {
+        opacity: 0.5
+        anchors.fill: parent
+        vendorExtensionsEnabled: false
+
+        // polar axis
+        ShapePath {
```

Comment:
We will require the auto increment adjustment / dynamic scaling like `DepthAxis`: https://github.com/bluerobotics/ping-viewer/blob/master/qml/DepthAxis.qml#L16

Reply:
Check https://github.com/bluerobotics/ping-viewer/pull/517#discussion_r293353491.